.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,7 +35,7 @@ Add as many details as possible to help others reproduce the issue and test the
 
 -------------------
 
-- [ ] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
+- [ ] Run `./vendor/bin/changelogger add` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
```

Comment:
This is fine. Maybe it would be a good idea to add NPM command inside `package.json`. For example, `npm run changelog` would execute the following command. The same principle was used for different scripts/actions in WooCommerce Payments

Reply:
Thanks for suggesting this. 👍 Addressed here cf41b6e01bb74d5a02c0de290726237d24a35d2b